designer/server/src/views/file/download-page.njk

```diff
@@ -0,0 +1,38 @@
+{% extends 'layouts/page.njk' %}
+
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+
+{% block content %}
+  <form method="post" novalidate>
+    {% call appPageBody({
+      heading: pageHeading,
+      errorList: errorList
+    }) %}
+
+    <p class="govuk-body-l">
```

Comment:
This is `govuk-body` in the prototype. `govuk-body-l` looks a bit off.

Reply:
done